meson.build

```diff
@@ -31,6 +31,7 @@ endif
 with_coverage = get_option('with_coverage')
 if with_coverage == 'true'
    add_project_arguments('-coverage', language : 'c')
+   add_project_arguments('-fprofile-dir=/var/tmp/bluechi-coverage', language : 'c')
```

Comment:
You should not use hardcoded path, but rather using specific options for directories:

```
 add_project_arguments(
    '-fprofile-dir=' + join_paths(prefixdir, get_option('localstatedir'), 'tmp', 'bluechi-coverage'), 
    language : 'c')
```